### PR TITLE
Set concurrency limit via environment variable

### PR DESCRIPTION
**Pull request by carlochess**
Currently, the concurrency limit for the cron job is set to 2 workers.
Since in some cases some users don't not need to use too many
resources and can get the job done wih only one worker,
this number could be set using an environment variable.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -25,15 +25,17 @@ if (fs.existsSync(jsonPath)) {
 		host: env('HOST', jsonConfig.host),
 		port: Number(env('PORT', jsonConfig.port)),
 		cron: env('CRON', jsonConfig.cron),
-		chromeLaunchConfig: jsonConfig.chromeLaunchConfig || {}
+		chromeLaunchConfig: jsonConfig.chromeLaunchConfig || {},
+		numWorkers: Number(env('NUM_WORKERS', '2'))
```
Comment (joeyciechanowicz) [inline config.js]: This should be `jsonConfig.numWorkers || 2`
```suggestion
		numWorkers: jsonConfig.numWorkers || 2
```

Comment (carlochess) [reply config.js]: Sure, I will.

**Comment by carlochess** [comment]
The changes have been applied 😄

**Comment by joeyciechanowicz** [comment]
Apologies for the delay in getting to this @carlochess . This is all great work and ready to be merged, there are just merge conflicts that have to be resolved first.

**Comment by carlochess** [comment]
@joeyciechanowicz The changes have been applied

**Pull request merged**